### ISSUE TITLE
problem running demo 

### ISSUE THREAD

**Issue by qian0001** [parklab/SCAN2#8]
When I ran the demo on a 20 core, 768GB computer I got the the following message before aborting:
...
Loading required package: rtracklayer
Loading required package: BSgenome.Hsapiens.1000genomes.hs37d5
Loading required package: SigProfilerMatrixGeneratorR
Error: UnboundLocalError: local variable 'indel_tsb_dict_abbrev' referenced before assignment
Execution halted

scan2 validate did not show errors.

What could be wrong?

**Comment by PreshitaDave** [comment]
Yes I get the exact same error! What is the solution? Please help!
